client/layout/sidebar/item.jsx

```diff
@@ -41,6 +44,19 @@ export default class SidebarItem extends React.Component {
 		}
 	};
 
+	componentDidMount() {
+		const { toggleSection } = this.props;
+
+		// props.sectionIsExpanded is initialized as `null`, which means it's not yet expanded.
```

Comment:
This seems like an oversight in the state implementation.

It seems like we want to default to `false` (collapsed). The simplest change seems use `false` as the default here:

https://github.com/Automattic/wp-calypso/blob/f7d06c6d5904a74e60654338dffb8de6f1223134/client/state/my-sites/sidebar/reducer.js#L17